ladybug_charts/to_figure.py

```diff
@@ -148,7 +148,81 @@ def monthly_barchart(data: MonthlyCollection,
             'xanchor': 'center',
             'yanchor': 'top'}
     )
+    fig.update_xaxes(showline=True, linewidth=1, linecolor="black", mirror=True)
+    fig.update_yaxes(showline=True, linewidth=1, linecolor="black", mirror=True)
+
+    return fig
+
+
+def barchart(data: List[MonthlyCollection],
```

Comment:
Hi @devngc! I think it will be more consistent with how we name the methods across the LBT libraries if we use `bar_chart` instead of `barchart`.

Reply:
Done